README.md

```diff
@@ -1,34 +1,54 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Next.js Starter ButterCMS
 
-## Getting Started
+Next.js starter project using ButterCMS. The project uses the sample content that is automatically created when you
+create your free trial account on ButterCMS.
 
-First, run the development server:
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2FButterCMS%2Fnextjs-starter-buttercms)
 
+- Demo: https://nextjs-starter-buttercms.vercel.app/
```

Comment:
What do you think about like, "Live Demo" or "See the Demo" or something? (e.g., something with more description or a call to action of sorts?